octobot/src/assets/users.html

```diff
@@ -34,7 +34,7 @@ <h4 class="modal-title">{{!!theUser.id ? 'Edit' : 'Add'}} user</h4>
             <input id="add-user-username" type="text" class="form-control" ng-model="theUser.github" placeholder="GitHub username" required>
           </div>
           <div class="form-group">
-            <input type="text" class="form-control" ng-model="theUser.slack" placeholder="Slack username" required>
```

Comment:
Unrelated to this PR, but reading this reminded me:

If you're going to edit an existing user, you don't see the placeholders, and it isn't clear which field is which. You have to delete the values to figure out what the fields stand for. I guess it will be more obvious now that one is an email though.

Reply:
hmm. true. hopefully everyone sees the placeholders?